src/app/shared/car-search/car-search.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component, Input} from '@angular/core';
+import {CarsService} from "../../services/cars-service";
+import {ICar} from "./car-model"
+import {FormControl, Validators} from "@angular/forms";
+import {Observable} from "rxjs";
+
+@Component({
+  selector: 'app-car-search',
+  templateUrl: './car-search.component.html',
+  styleUrls: ['./car-search.component.scss']
+})
+export class CarSearchComponent {
+  @Input () placeholder: string = '';
+
+  cars$:Observable<ICar[]> = new Observable<ICar[]>();
+
+  searchText: FormControl = new FormControl('',
```

Comment:
why do you need that validation?
you need 'required' validation, if input is empty or when it's toched or is reseted